st2chatops.env

```diff
@@ -31,6 +31,12 @@ export ST2_STREAM_URL="${ST2_STREAM_URL:-https://${ST2_HOSTNAME}/stream}"
 # StackStorm API key
 export ST2_API_KEY="${ST2_API_KEY}"
 
+# Stackstorm Token
+# When authentication is enabled (Uncomment ST2_AUTH_URL, ST2_AUTH_USERNAME,
```

Comment:
Looking a few lines above, `ST2_AUTH_URL` env is already uncommented ^^.

I think the entire message stands out with the existing minimalistic code comments.

Can we simplify it to one comment line, placing after `# ST2 credentials. Fill in to use any stackstorm account.`?
Stating something along the lines that st2 username:password pair will rely on authentication token re-generation.